pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -38,39 +39,49 @@ func NewPerfCounterClient() (Client, error) {
 
 // perfCounterNodeStatsClient is a client that provides Windows Stats via PerfCounters
 type perfCounterNodeStatsClient struct {
-	nodeStats
-	mu sync.RWMutex
+	nodeMetrics
+	mu sync.RWMutex // mu protects nodeMetrics
+	nodeInfo
 }
 
 func (p *perfCounterNodeStatsClient) startMonitoring() error {
 	memory, err := getPhysicallyInstalledSystemMemoryBytes()
+
 	if err != nil {
 		return err
 	}
-	p.nodeStats.memoryPhysicalCapacityBytes = memory
 
 	version, err := exec.Command("cmd", "/C", "ver").Output()
 	if err != nil {
 		return err
 	}
-	p.kernelVersion = strings.TrimSpace(string(version))
 
-	cpuChan, err := readPerformanceCounter(cpuQuery)
+	kernelVersion := strings.TrimSpace(string(version))
+	p.nodeInfo = nodeInfo{
+		kernelVersion:               kernelVersion,
+		memoryPhysicalCapacityBytes: memory,
+		startTime:                   time.Now(),
```

Comment:
@yguo0905 do you remember whether kubelet uses this timestamp?

Reply:
We use it in the summary API here, but not sure if/how downstream uses it.

https://github.com/kubernetes/kubernetes/blob/58b126d98bd1867678ef3dd5c46db1b15d9a1fb8/pkg/kubelet/server/stats/summary.go#L75

Reply:
Is this the root cgroup creation time in cadvisor's implementation? 
I guess either way it's not very meaningful...

Reply:
It used is by heapster to calculate uptime as well: https://github.com/kubernetes/heapster/blob/master/metrics/sources/summary/summary.go#L247

Reply:
The current implementation is consistent with the description of the field in the summary API, so I think it's good for now.
https://github.com/kubernetes/kubernetes/blob/v1.9.0-alpha.0/pkg/kubelet/apis/stats/v1alpha1/types.go#L41